tests/basic/storage_pool_test.go

```diff
@@ -141,33 +144,31 @@ var _ = Describe("{StoragePoolExpandDiskAdd}", func() {
 			enterPoolMaintenanceAddDisk(poolIDToResize)
 			defer exitPoolMaintenance(poolIDToResize)
 
-			err = Inst().V.ExpandPool(poolIDToResize, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, false)
-			if err != nil {
-				isPoolAddDiskSupported := IsPoolAddDiskSupported()
-				if !isPoolAddDiskSupported {
-					IsExpectederr := strings.Contains(err.Error(), "add-drive type expansion is not supported with px-storev2. Use resize-drive expansion type")
-					dash.VerifyFatal(IsExpectederr, true, err.Error())
-					log.InfoD("Drive add not supported :%s, hence skipping the test", err.Error())
-					Skip("drive add to existing pool not supported for px-storev2 or px-cache pools")
-
+			if IsPoolAddDiskSupported() {
+				err = Inst().V.ExpandPool(poolIDToResize, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, false)
+				if err != nil {
+					log.FailOnError(err, "is Pool Expand using Add disk successful ?")
```

Comment:
both log.FailonError and dash.VerifyFatal are not required. one is enough

Reply:
my bad !! fixed